packages/ozone/src/db/migrations/20240710T051104136Z-sets.ts

```diff
@@ -0,0 +1,63 @@
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  // Create the sets table
+  await db.schema
+    .createTable('ozone_set')
```

Comment:
I assume `ozone_` is prefixed here to avoid naming the table the reserved word "set".  There could be other ways that feel natural to name the tables and still avoid the reserved word, e.g. `set_detail` and `set_item`.